Java-Control-Statements.md

```diff
@@ -0,0 +1,223 @@
+#Java Control Statements :
+
+In any programming language, the control over normal flow of program is necessary to make dynamic decisions at runtime and achieving desired results. `Control statements` manage the flow of execution of the program and decide the branching of logical flow. Java's program control statements can be put into following categories: `selection` , `iteration` and `jump statement` . `Selection statement` allows a program to choose different path of executions depending on an expression. `Iteration` allows a program to repeat one or more statements. `Jump statement` is used to transfer control to another part of the program. As `iteration` and `jump statements` are already covered in `Java-loops`, here we would discuss `Selection statements`.
+
+##Java's selection statement :
+
+`Selection statements` allow a program to choose different executions based on the outcome of an `expression` or `state of a variable
+`.
+Java has two selection statements : `if-else` and `switch`.
+
+###if-else
+
```

Comment:
Usually people write `if...else` statement instead of `if-else`. (*just nitpicking, no big deal*)